subsys/bluetooth/services/fast_pair/fp_keys.c

```diff
@@ -155,35 +160,39 @@ static int key_gen_public_key(struct bt_conn *conn, struct fp_keys_keygen_params
 	return err;
 }
 
-static int key_gen_account_key(struct bt_conn *conn, struct fp_keys_keygen_params *keygen_params)
+static bool key_gen_account_key_check(const uint8_t *account_key, void *context)
 {
 	int err;
+	uint8_t req[FP_CRYPTO_AES128_BLOCK_LEN];
+	struct fp_key_gen_account_key_check_context *ak_check_context =
```

Comment:
The cast from `void *` is not needed. Please remove it

Reply:
fixed